src/zkevm_specs/evm/execution/copy_to_log.py

```diff
@@ -0,0 +1,46 @@
+from ...util import FQ, N_BYTES_MEMORY_SIZE
+from ..execution_state import ExecutionState
+from ..instruction import Instruction, Transition
+from ..step import CopyToLogAuxData
+from ..table import RW, TxLogFieldTag
+from ..util import BufferReaderGadget
+
+
+MAX_COPY_BYTES = 71
```

Comment:
Why is it 71 bytes? Can we comment this?

Reply:
it the same to `memory_copy` , move it to param.py  and make it common use for both `copy_to_log`&`memory_copy` in [87f20da](https://github.com/appliedzkp/zkevm-specs/pull/125/commits/87f20da4503b3f17e7fc865186616092570d9415)